### PR TITLE
Added verida custom button

### LINKED ISSUE #65: Add verida connect button from [account-web-vault]  package
# Task 

- The `account-web-vault` packages  should support adding the verida connect button image to third parties application, 
this can be done by adding custom classnames and using the `window.onload` to inject the img to the DOM

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added verida custom button,  so that developers can pass and id `verida-button` to an html input tag 

resolves #65

### CODE REVIEW

**Comment by nick-verida** [comment]
I'm not able to give any useful feedback on this, sorry. I don't know TS or how the library works. 

Happy to verify it works if there is a test harness, but beyond that I don't think I'm useful.

@tahpot will need to review.

**Comment by tahpot** [comment]
@cmcWebCode40 

Any reason `addVeridaButtonImage()` is called three times?

I would have thought the image would be injected once when the page loads?

**Comment by cmcWebCode40** [comment]
> @cmcWebCode40
> 
> Any reason `addVeridaButtonImage()` is called three times?
> 
> I would have thought the image would be injected once when the page loads?

ok, I noticed that the verida button image gets removed from the DOM whenever the modal is closed.
what way I can resolve this to call the function once?

**Comment by cmcWebCode40** [comment]
> addVeridaButtonImage()

@tahpot  here is a short demo ,

https://www.loom.com/share/fb97130c9cfc4018acf56510a9785e9f

i called the function once on-load ,but whenever i open and close the QR modal the button unmount , that was why i called the addVeridaButtonImage() , when the modal is clicked .

**Comment by tahpot** [comment]
As discussed, let's undo this and just provide a static image asset for developers to copy and past and include that in the docs.

**Comment by cmcWebCode40** [comment]
> As discussed, let's undo this and just provide a static image asset for developers to copy and paste and include that in the docs.

@taphot, this PR has been updated with the requested changes.

File: packages/account-web-vault/README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
+# Add Custom verida button
 
-@todo
\ No newline at end of file
+Here you can add our custom verida image url in your application so as to main our brand across application see example below.
+
+```
+<img src="https://assets.verida.io/connect-button.png" alt="Verida Vault App" />
```
Comment (tahpot) [inline packages/account-web-vault/README.md:6]: @cmcWebCode40 @nick-verida 

`https://assets.verida.io/connect-button.png` does not exist

**Review by tahpot — CHANGES_REQUESTED** [review]

**Comment by cmcWebCode40** [comment]
@tahpot , thanks

changes requests noted
i will resolve them

**Comment by nick-verida** [comment]
@cmcWebCode40 do you know what the status of this is?

**Comment by nick-verida** [comment]
@cmcWebCode40 @tahpot Does this need merging?

**Comment by nick-verida** [comment]
Looks like this is redundant.

**Pull request closed without merging**